### DISCUSSION TITLE
Is it possible to use a custom local (i.e.: `en-int`)

### DISCUSSION THREAD

**Discussion by fpoirier1** [amannn/next-intl#1581] [Q&A]
My client is requesting having a locale called `en-int` that represent "all the other countries". 

I was able to configure next-intl like so : 

```
export const routingConfig = defineRouting({
  locales: ['en-us', 'en-ca', 'fr-ca', 'en-zw', 'en-int'],
  localePrefix: 'as-needed',
  defaultLocale: 'en-us',
});
```

Which works good until I need to format messages. 

`t('readtime.minutes', { count: readTime.minutes })` or 

```
  const format = useFormatter();
  const dateTime = new Date(date);
  const formattedDate = format.dateTime(dateTime, {
    year: 'numeric',
    month: 'long',
    day: 'numeric',
  });
```

I get the error message `IntlError: INVALID_MESSAGE: Incorrect locale information provided`

It's important to note that I am relying on the locale `en-int` to do my API calls to my CMS.

Thank you in advance !

**Comment by amannn** [comment]
I think you're looking for this: https://next-intl-docs.vercel.app/docs/routing#locale-prefix-custom
